### DISCUSSION TITLE
Den Preis von schweren Waffen Teams erhöhen

### DISCUSSION THREAD

**Discussion by Bellum45** [dstahlmann/cold-war-crisis#36] [Ideas]
Ich finde das der Preis von Cal. 50 Teams, M47 Dragon Teams, ATGM Teams, DShK 12,7mm MG Teams und RPG auf Lafette zu niedrig.

Ich finde sie für den aktuellen Preis sind sie zu effizient in ihren Job.

Aus dem Grund haben David und ich, solange wie wir Modelle hatten die schweren Waffen Teams genommen.

**Comment by dstahlmann** [comment]
Fritz und ich sind gestern zum selben Ergebnis gekommen. Gerade Dragon/ATGM-Teams sind für T1 zu stark oder zu günstig. Fritz schlug vor, diese auf T2 zu setzen, ich denke der Preis sollte das eher Regeln. Dann sind die Verfügbar, aber eben nicht in der Masse.

**Reply by Bellum45** [reply]
Ich vermute du meinst T2

**Reply by dstahlmann** [reply]
Selbstverständlich. Sorry! Ist editiert.
